.github/workflows/system-tests-enterprise.yml

```diff
@@ -161,15 +151,16 @@ jobs:
     # When increasing the timeout make sure it's not larger than the schedule cron interval
     timeout-minutes: 360
     name: Run System Tests Enterprise
-    runs-on: ubuntu-latest
+    # requires prepare to finish before starting
     needs: [prepare-system-tests-enterprise-ci]
+    runs-on: ubuntu-latest
+    # let's not run this on every fork, change to your fork when developing
+    if: github.repository == 'mlrun/mlrun' || github.event_name == 'workflow_dispatch'
+    continue-on-error: true
     strategy:
       max-parallel: 1
       matrix:
         test_component: [api,runtimes,feature_store,projects,model_monitoring,examples,demos,backward_compatibility]
```

Comment:
Use:
```
strategy:
  fail-fast: false
```
You won't need that last step  `verify-all-system-tests-succeeded` as if one of the matrix steps fails, the job will fail but all the matrix steps will run. and `continue-on-error` is mainly for the next jobs (of which there aren't any), not within the matrix. See:
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategyfail-fast